### ISSUE TITLE
Multi-Character emojis do not render correctly

### ISSUE THREAD

**Issue by cmjacobs** [toptensoftware/RichTextKit#8]
Emoji sequences that are made up of multiple characters combined with the Zero-Width joiners are getting rendered as individual emoji, not the resulting single emoji. A good example of this is the pirate flag (🏴‍☠️) is getting rendered as a black flag (🏴) and a skull and crossbones (☠).

On line 79 of FontFallback.cs, there is a comment:
```csharp
// Consume as many characters as possible using the requested type face
```
However, on line 80, there is a 1 hard-coded and the call to consume additional characters is commented out:
```csharp
count = 1;// RunFace.GetGlyphs((IntPtr)(pch + pos), length - pos, SKEncoding.Utf32, out var glyphs);
```

If I remove the 1 and restore the call to ```RunFace.GetGlyphs```, the emoji render correctly. 
I am curious why the 1 was hard-coded and what possibly breaks by restoring the commented out code.

**Comment by toptensoftware** [comment]
Hi @cmjacobs 

Thanks for reporting... I can't recall off hand why that change was made but I'll look into it when I get a chance.

Brad

**Comment by toptensoftware** [comment]
I've put in a fix for this in build 119 (available on nuget now).  I'm not sure it's a completely accurate fix, because the heuristics on when to switch back from a fallback font aren't clear to me.  But seems to work better.

**Comment by toptensoftware** [comment]
There was a regression in build 0.3.129 that caused this issue to come back.  It's been fixed again in 0.3.130.
